src/external/spglib.jl

```diff
@@ -88,26 +99,49 @@ spglib_cell(lattice, atoms) = first(spglib_cell_atommapping_(lattice, atoms))
     Stildes, τtildes
 end
 
+function spglib_standardize_cell_(unit_cell, to_primitive, no_idealize, symprec)
```

Comment:
I don't like that there is a function with and without underscore. Can't you just merge them?